app/views/archives/_archive_controls.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="form-group-columns">
+  <div class="well well--dark">
+    <p class="mt-0">You may choose to transfer this post to yourself.</p>
```

Comment:
Please add headings to these columns. The headings might be the exact same as the buttons but it might make it easier to see what's going on at first glance